src/device.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async function device(req, res) {
+  let [gaiaCommit] = await Promise.all([
```

Comment:
Yep, since I plan on adding `adb shell getprop` and `getGeckoRevision` here as well.